laurent22/joplin
# Mobile: Plugins: Show information page before enabling plugin support

# Summary

This PR shows an information page before plugin support is enabled.


# Screenshots


<img src="https://github.com/laurent22/joplin/assets/46334387/e2665fbe-5cd8-4c55-8e98-81e608b01a99" alt="screenshot: settings section: Plugins: What are plugins? \\ Plugins extend Joplin with features that are not present by default. Plugins can extend Joplin's editor, viewer, and more. ((continued))" width="300"/><img src="https://github.com/laurent22/joplin/assets/46334387/f82fe01e-e8a3-4287-9e66-fe2e225029be" alt="screenshot: settings section: Scrolled down: Plugins: Like any software you install, plugins can potentiallhy cause security issues or data loss. Newline. Here's what we do to make plugins safer. Card: Icon: Crown: Recommended plugins: Body: We mark plugins developed by trusted Joplin community members as quote recommended unquote. Card: Icon: Branch: Open Souruce: Most plugins have source code freely available for review on the plugin website. Card: Report system: We have a system for reporting and removing problematic plugins // learn more button // Enable plugin support button" width="300"/>

# Testing plan

1. Navigate to settings > plugins.
2. Verify that the plugin about screen is shown.
3. Click "Learn more".
4. Verify that the plugin help page is opened.
5. Go back to Joplin.
6. Click "Enable plugin support".
7. Verify that the existing plugin screen is shown and all plugins are loaded (if any were installed before testing the PR).
8. Install the note tabs plugin.
9. Verify that the "show plugin panels" icon is visible in the toolbar.
10. Click "show advanced settings" and use the "enable plugin support" toggle to disable plugin support.
11. Verify that the "show plugin panels" icon disappears.

This has been tested successfully on an Android 10 emulator.
